test/bench_internal/parser_benchmarks.hpp

```diff
@@ -0,0 +1,15 @@
+// Fizzy: A fast WebAssembly interpreter
```

Comment:
Maybe this file should be `experimental.hpp`, because two of its three functions are defined in `experimental.cpp`?

Also not sure if the third one - `leb128u_decode_u64_noinline` - needs to be in its own cpp.

Reply:
I would go with something less specific than `parser_benchmarks.cpp`: `experimental.cpp` is fine.

This is not explicitly needed now as I discovered `[[noinline]]` attribute...

Reply:
This is resolved now to use `experimental.hpp` for alternative LEB128 implementation. The other functions were moved to `parser_benchmarks.cpp` with `[[gnu::noinline]]` attribute.